js/modules/product-info.js

```diff
@@ -0,0 +1,47 @@
+import { api } from "./api.js";
+
+const titlesInformation = document.querySelectorAll("[data-information-title]");
+const informationText = document.querySelector(".information-text");
+const rangeValue = document.querySelector(".remaining-value");
+
+export default async function initInfoProduct() {
+  const response = await api.get(`/produto/1`);
+  const { meta_valor, valor_arrecadado, total_apoios, data_limite } =
+    response.data;
```

Comment:
cuidado com esse uso direto aqui de desestruturação, hoje você sabe que esses dados vem do back end, porém amanhã pode não vir mais, causando uma quebra, se vc colocar aquele `{}` no final, ele não vai quebrar e retornar todo o conteúdo que não existir como undefined
```suggestion
  const { meta_valor, valor_arrecadado, total_apoios, data_limite } =
    response.data || {};
```

Reply:
Resolvido